.circleci/workflows.yml

```diff
@@ -377,26 +374,16 @@ jobs:
             echo $TAG > /tmp/docker-tag.txt
       - run:
           name: Build & Publish Management API Docker Image to Azure Registry
+          when: always          
           command: |
             export AM_VERSION=$(mvn -s /tmp/.circleci.settings.xml -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
             echo "Gravitee AM version: ${AM_VERSION}"
             export IMAGE_TAG=graviteeio.azurecr.io/am-management-api:${TAG}
             echo "export IMAGE_TAG=$IMAGE_TAG" >> $BASH_ENV
             docker build -f docker/management-api/Dockerfile-dev --build-arg GRAVITEEAM_VERSION=${AM_VERSION} -t ${IMAGE_TAG} /tmp
-      - docker-scan
-      - run:
-          when: always
-          name: Generate last log lines for slack message
-          command: |
-            API_URL=$(echo $CIRCLE_BUILD_URL | cut -d/ -f4-7)
-            FAILED_STEP=$(curl "https://circleci.com/api/v1.1/project/${API_URL}?circle-token=${CIRCLE_API_TOKEN}" | jq '.steps | .[] | flatten | map(select(.name? == "Run Snyk")) | .[] | {allocation_id, step}')
-            ALLOCATION_ID=$(echo "${FAILED_STEP}" | jq -r '.allocation_id')
-            STEP=$(echo "${FAILED_STEP}" | jq -r '.step')
-            curl -o slack_response.txt "https://circleci.com/api/v1.1/project/${API_URL}/output/${STEP}/0?file=true&allocation-id=${ALLOCATION_ID}&circle-token=$CIRCLE_API_TOKEN"
-            sed -i 's/\r$//g' slack_response.txt
-            SLACK_MESSAGE=$(cat slack_response.txt | tail -n 15 | sed '$!s/$/\\n/' | tr -d '\n')
-            echo "export SLACK_MESSAGE='${SLACK_MESSAGE}'" >> $BASH_ENV     
-      - notify-snyk-result 
+            SLACK_MESSAGE1="--AM Docker Images-- ${IMAGE_TAG},"
+            echo "export SLACK_MESSAGE='${SLACK_MESSAGE1}'" >> $BASH_ENV   
+            snyk container test ${IMAGE_TAG} --app-vulns
```

Comment:
with this change, does it mean that the pipeline will fail here if there are some vuln issues ?
int the previous implementation using the snyk orb, we go to the next step in case of failure https://github.com/gravitee-io/gravitee-access-management/pull/1824/files#diff-7bf091450b804ea25dc3d11ac28c50202d19178f7b53f54a02ad59c946afa6a7L83

Reply:
This will go to next step even if image test fails.  This is using snyk cli as only cli has app vulnerability feature.  So in the step i added when : always which will run even previous step fails.